### PR TITLE
chore(polyfills): remove exports and entrypoints

### PR DESCRIPTION
**Pull request by Theeraphat-Sorasetsakul**
## Description
Current exports in polyfills conflicts with elf-doc content at `import '@refinitiv-ui/polyfills/minimal';`. That will got error.

> Error: Module not found: Error: Package path ./minimal is not exported from package 
\node_modules\@refinitiv-ui\polyfills (see exports field in \node_modules\@refinitiv-ui\polyfills\package.json)

Fixes # (issue)

## Type of change
Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by Theeraphat-Sorasetsakul** [comment]
I have test this on angular 11 and 12. Both are worked on IE.

**Comment by wsuwt** [comment]
Looks good to me 👍

File: packages/polyfills/package.json [hunk 1]
```diff
@@ -4,14 +4,7 @@
...
   "license": "Apache-2.0",
-  "main": "entrypoints/all.js",
-  "exports": {
-    ".": {
-      "default": "./entrypoints/all.js"
-    },
-    "./all.js": "./entrypoints/all.js",
-    "./minimal.js": "./entrypoints/minimal.js"
-  },
+  "main": "all.js",
```
Comment (wsuwt) [inline packages/polyfills/package.json:7]: @Theeraphat-Sorasetsakul What do you think if we change `all.js` into `index.js`. To me, it looks cleaner that way.

Comment (Theeraphat-Sorasetsakul) [reply packages/polyfills/package.json]: I think so too. It will look better. I will update it.

**Review by wsuwt — APPROVED** [review]

**Pull request merged**